apcd-cms/src/apps/exception/templates/exception_submission_form/exception_submission_error.html

```diff
@@ -12,7 +12,7 @@ <h1>Request to Submit Exception</h1>
     <p class="c-message c-message--error">
         An error occurred during your exception request submission. For help, <a href="/workbench/dashboard">submit a ticket</a>.
     </p>
-    <a class="c-button c-button--primary" href="/submissions/exceptions/">Go to Exception Forms</a>
+    <a class="c-button c-button--primary" href="/submissions/exceptions/">Go Back to Exception Form</a>
```

Comment:
```suggestion
    <a class="c-button c-button--primary" href="/submissions/exception/">Go Back to Exception Form</a>
```
Back button is broken